websites/M/marcuscodes.me/presence.ts

```diff
@@ -0,0 +1,32 @@
+const presence = new Presence({
+  clientId: "782532120047779850"
+});
+const timeStamp = Math.floor(Date.now() / 1000);
+const page = window.location.href;
```

Comment:
Merge consts

Reply:
sorry for my stupidity, what does that mean

Reply:
e.g.: ```const d = f(),
g = h();```

Reply:
i hope this works

Reply:
![image](https://user-images.githubusercontent.com/9348108/100627994-04f7f700-3328-11eb-8138-649bc15cf710.png)
you didnt change anything